### PR TITLE
Implement a fix in DataCorrectnessRoutesTest #255 and updated the backend README

### PR DESCRIPTION
**Pull request by peterstran**

### CODE REVIEW

File: backend/src/test/kotlin/blue/mild/covid/vaxx/routes/DataCorrectnessRoutesTest.kt [hunk 1]
```diff
@@ -79,6 +79,15 @@ class DataCorrectnessRoutesTest : ServerTestBase() {
         // by calling the API and requesting data for patient1.id and expecting Ok status
         // and then comparing correctnessId with the ID you received in the test
         // hint: see first part of the test
+
```
Comment (tomaskourim) [inline backend/src/test/kotlin/blue/mild/covid/vaxx/routes/DataCorrectnessRoutesTest.kt:82]: you can remove the TODO and the instructions part of the text, just leave the description

File: backend/src/test/kotlin/blue/mild/covid/vaxx/routes/DataCorrectnessRoutesTest.kt [hunk 2]
```diff
@@ -79,6 +79,15 @@ class DataCorrectnessRoutesTest : ServerTestBase() {
...
+        handleRequest( HttpMethod.Get, "${Routes.dataCorrectness}?id=${patient1.id}" ) {
```
Comment (tomaskourim) [inline backend/src/test/kotlin/blue/mild/covid/vaxx/routes/DataCorrectnessRoutesTest.kt:83]: use some autoformat (Idea has it built in, Ctrl+Alt+L) to remove extra spaces and insert the missing ones

File: backend/src/test/kotlin/blue/mild/covid/vaxx/routes/DataCorrectnessRoutesTest.kt [hunk 3]
```diff
@@ -79,6 +79,15 @@ class DataCorrectnessRoutesTest : ServerTestBase() {
...
+            authorize()
+        }.run {
+            expectStatus(HttpStatusCode.OK)
+            val output = receive<DataCorrectnessConfirmationDetailDtoOut>()
+            print(assertEquals(correctnessId,output.id))
```
Comment (tomaskourim) [inline backend/src/test/kotlin/blue/mild/covid/vaxx/routes/DataCorrectnessRoutesTest.kt:88]: why do you need to print it?

**Review by tomaskourim — CHANGES_REQUESTED** [review]

**Review by tomaskourim — APPROVED** [review]

**Comment by LukasForst** [comment]
@peterstran feel free to merge

* closes #255

**Pull request merged**